src/types.d.ts

```diff
@@ -0,0 +1,88 @@
+
+
+declare namespace NodeJS {
```

Comment:
This seems a little weird to me.  Are these types on other imported modules?  If so, shouldn't we be making PRs to DefinitelyTyped?

Reply:
Since logging has not been migrated yet, we need to explicitly define the types for it. Once it has been migrated, we can just use them directly.